packages/babel-plugin-ima-logger/jest.config.js

```diff
@@ -0,0 +1,15 @@
+const base = require("../../jest.config.base.js");
+
+module.exports = {
+    ...base,
+    globals: {
+        functions: 10,
+        lines: 10,
+        statements: 10
+    },
+    testEnvironment: "node",
+    setupFiles: [],

```

Comment:
There is no need to define empty setupFiles.

Reply:
Removed.